apps/native/app/src/screens/wallet-pass/wallet-pass.tsx

```diff
@@ -349,6 +349,7 @@ export const WalletPassScreen: NavigationFunctionComponent<{
           nativeID={`license-${licenseType}_destination`}
           type={licenseType}
           logo={
+            isBarcodeEnabled &&
```

Comment:
@eirikurn @snaerseljan

1. I could not find any Gervimaður that has a photo for his driving license so I could not verify 100% that this works.

2. There is also a variable called allowLicenseBarcode that is the following:
```
const allowLicenseBarcode = isBarcodeEnabled && (data?.license?.pkpass && data?.license?.pkpassStatus === GenericUserLicensePkPassStatus.Available) && !data?.payload?.metadata?.expired
```

Just wanted to make sure that we don't want to use that, I only used the feature flag since that is was noted in the Notion task 😄

Reply:
Yes this is sufficient :)